### ISSUE TITLE
Expand ou:alias regex to allow "_" underscore

### ISSUE THREAD

**Issue by e2-ibm** [vertexproject/synapse#427]
```
('ou:alias',
                 {'subof': 'str:lwr', 'regex': '^[0-9a-z]+$', 'doc': 'An alias for the org GUID', 'ex': 'vertexproj'})
```

E Comments:
1.  I’d like to be able to use a delimiter that is a character that is also allowable in tag names (Currently I can make a tag named #pwn.foogov_mfa so I can pivot between syn:tag:base and ou:org:alias)
2. I'd like to be able to distinguish between the two orgs: “Saudi Arabia MFA” and “Turkey MFA” via my ou:org:alias (e.g. I can't have both be named mfa)

thesilence Comments:
1. Orgs have always had a "special" role in tags due to the desire to reference an entity (org node) and an analytical observation associated with that entity (tag) with the same string (e.g., ou:org:alias=google, legit.google, etc.
From a design standpoint, visi has talked about (eventually) allowing the use of *guids* in tags instead of the org string - basically you'd have "legit.<guid_for_google>" that would get visually translated (represented) using the alias for that org.
Which (eventually) helps with tag management so you don't have to manually enforce / check string consistency blah blah

2. From a "what is an alias" standpoint, it's really just humon-usable shorthand to refer to a thing without having to use its guid
From that perspective, I don't think (?) the "allowable characters" matter as long as people understand their own "shorthand" syntax for creating them
(problems arise if different instances of synapse use different aliases and want to share data, but that's a separate issue)

epiphyte 

if we expand something like ou:alias and someone had a sepr type using it and we added THEIR sepr character to the allowed regexes they could then make un-normable sepr primary property.

thesilence

i def agree with you on model creep - that's why these discussions are helpful, bounce around dem unintended consequences

so...thinking of how we use aliases, I want to:

1. use a humon-usable string i can use to reference a type of object i use all the time that is otherwise represented by a not-humon-friendly GUID (so I can type fewer characters and use something i can remember easily)

2. have some way (via ou:alias or other props) to locate / identify an object within synapse that is defined by a GUID and by largely arbitrary strings (like an org - i'm not sure of the alias but I need to see if this thing exists and find it if it does)

we always dealt with 2. by searching arbitrary strings / substrings ("find me stuff that contains "ministry" and "saudi") and / or country codes ("find me things with CC = SA")

Whether use of an underscore (or other character) helps, hinders, or is indifferent to that is debatable I suppose.

Like...is "sagovmfa" as an alias too awful to type? If you didn't know the alias, would it be really hard to find the org node for "Saudi Arabian Ministry of Foreign Affairs" if you could do a reasonable string search to find the org and therefore the alias?

(No good answer, just being philosophical)

**Comment by invisig0th** [comment]
We can confirm that this is the case in the new 0.1.x release.  If not, it'll go into the next release (within 2 weeks)

**Comment by vEpiphyte** [comment]
@e2-ibm This will be included in the next synapse release (v0.1.18).
